prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -1,120 +1,167 @@
-kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
-metadata:
-  name: weave-net
-rules:
-- apiGroups:
-  - ""
-  resources:
-  - pods
-  - namespaces
-  - nodes
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - extensions
-  resources:
-  - networkpolicies
-  verbs:
-  - get
-  - list
-  - watch
----
 apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: weave-net
-  namespace: kube-system
----
-kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1beta1
-metadata:
-  name: weave-net
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: weave-net
-subjects:
-- kind: ServiceAccount
-  name: weave-net
-  namespace: kube-system
----
-apiVersion: extensions/v1beta1
-kind: DaemonSet
-metadata:
-  name: weave-net
-  namespace: kube-system
-spec:
-  template:
-    metadata:
+kind: List
+items:
+  - metadata:
       labels:
         name: weave-net
+      name: weave-net
+      annotations:
+        cloud.weave.works/launcher-info: |-
```

Comment:
The idea with these files was to keep them as close as what we'd get using the Launch Generator in order to spot issues. These annotations shouldn't change anything in terms of behaviour though, so we could indeed remove/change them. 
What is your concern? Do you think these could be misleading and/or create confusion?